src/rpdk/core/contract/suite/handler_commons.py

```diff
@@ -168,6 +171,13 @@ def test_input_equals_output(resource_client, input_model, output_model):
     # only comparing properties in input model to those in output model and
     # ignoring extraneous properties that maybe present in output model.
     try:
+        transform_model(
```

Comment:
since this is so dependent on the resource client might it make sense to have it as a method on the client? could probably reduce the number of arguments then and get rid of transform keys, property transform, and jinja template